### PR TITLE
[NUnit] Add rerun command to the test pad.

### PR DESCRIPTION
**Pull request by Therzok**

### CODE REVIEW

**Comment by mrward** [comment]
Code changes look good to me.

Only minor thing is that we have "Re-run test" in the context menu but "Rerun Tests" for the button in the Test Results pad. So I guess we should use Rerun in the new context menu label.

**Comment by Therzok** [comment]
Sure thing. I can relabel it. I think we need to redesign the NUnit service to support not-reloading the whole test results when triggering a rerun.

File: main/src/addins/NUnit/Gui/TestResultsPad.cs [hunk 1]
```diff
@@ -604,9 +604,19 @@ protected void OnShowTestCode ()
 			if (loc != null)
 				IdeApp.Workbench.OpenDocument (loc.FileName, loc.Line, loc.Column);
 		}
+
+		[CommandHandler (TestCommands.RerunTest)]
+		protected void OnRerunTest ()
+		{
+			UnitTest test = GetSelectedTest ();
```
Comment (DavidKarlas) [inline main/src/addins/NUnit/Gui/TestResultsPad.cs]: Is SelectedTest==Last ran test?

Comment (Therzok) [inline main/src/addins/NUnit/Gui/TestResultsPad.cs]: No, it's the test you select in the results pad.

**Comment by DavidKarlas** [comment]
We got button for re-running already in result page...
Shouldn't logic/code be unified?

**Comment by Therzok** [comment]
@DavidKarlas That reruns with the _root_ unit test. Possibly the code can be deduplicated, yes.

**Pull request merged**